src/app/workspace/[workspaceId]/channel/[channelId]/layout.tsx

```
@@ -32,6 +33,519 @@ import { useWorkspaceId } from "@/hooks/use-workspace-id";
+					<VisuallyHidden.Root>
+						<DialogDescription>
+							Rename this channel to match your case.
+						</DialogDescription>
+					</VisuallyHidden.Root>
+				</DialogHeader>
+
+				<form className="space-y-4" onSubmit={onSubmit}>
+					<div className="space-y-4">
+						<div className="flex flex-col gap-2">
+							<div className="flex items-center justify-between">
+								<label className="text-sm font-medium">Channel Icon</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)